From d533f29d4a6dd77ba74f0fa3d71f63c117229570 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Wed, 9 Oct 2019 23:43:28 +0000
Subject: [PATCH 605/830] ANDROID: gki_defconfig: Add qcom pcie options to
 gki_defconfig
X-NVConfidentiality: public

The qcom PCIE support isn't able to be added as a module, so I
propose enabling CONFIG_PCIE_QCOM and CONFIG_PCIEPORTBUS in
the gki_defconfig.

RAS falls out, as when CONFIG_PCIEPORTBUS=y PCIEAER defaults
to =y, which selects RAS

For consistency, this also adds CONFIG_PCIEPORTBUS=y to x86
gki_defconfig

Change-Id: I04941e17f660dd13b2ef2ab3150acae3ce670b19
Bug: 146449535
Signed-off-by: John Stultz <john.stultz@linaro.org>
---
 arch/arm64/configs/gki_defconfig | 3 ++-
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 0023ec534144..d105c6765be2 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -206,7 +206,9 @@ CONFIG_MAC80211=y
 # CONFIG_MAC80211_RC_MINSTREL is not set
 CONFIG_RFKILL=y
 CONFIG_PCI=y
+CONFIG_PCIEPORTBUS=y
 CONFIG_PCI_HOST_GENERIC=y
+CONFIG_PCIE_QCOM=y
 CONFIG_PCIE_KIRIN=y
 CONFIG_FW_LOADER_USER_HELPER=y
 CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
@@ -389,7 +391,6 @@ CONFIG_IIO=y
 CONFIG_PWM=y
 CONFIG_QCOM_PDC=y
 CONFIG_GENERIC_PHY=y
-CONFIG_RAS=y
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_INTERCONNECT=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index c9cfd24e913d..1e10cf98ba1e 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -182,6 +182,7 @@ CONFIG_MAC80211=y
 # CONFIG_MAC80211_RC_MINSTREL is not set
 CONFIG_RFKILL=y
 CONFIG_PCI=y
+CONFIG_PCIEPORTBUS=y
 CONFIG_FW_LOADER_USER_HELPER=y
 CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
 # CONFIG_FW_CACHE is not set
-- 
2.17.1

